consent-ws/src/main/java/org/broadinstitute/consent/http/service/EmailNotifierService.java

```diff
@@ -141,7 +134,7 @@ public void sendCollectMessage(Integer electionId) throws MessagingException, IO
             Map<String, String> data = retrieveForCollect(electionId);
             String collectUrl = generateCollectVoteUrl(SERVER_URL, data.get("electionType"), data.get("entityId"), data.get("electionId"));
             Writer template = templateHelper.getCollectTemplate(data.get("userName"), data.get("electionType"), data.get("entityName"), collectUrl);
-            List<String> emails = StringUtils.isNotEmpty(data.get("additionalEmail")) ? Arrays.asList(data.get("additionalEmail"), data.get("email")) : Collections.singletonList(data.get("email"));
+            Set<String> emails = StringUtils.isNotEmpty(data.get("additionalEmail")) ? new HashSet<>(Arrays.asList(data.get("additionalEmail"), data.get("email"))) : new HashSet<>(Collections.singletonList(data.get("email")));
```

Comment:
All of the `new HashSet<>(Collections.singletonList(data.get("email")))` should be replaceable with `Collections.singleton(data.get("email"))`

See https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#singleton-T-

Reply:
Done!  Thanks for your comments were really helpful.